.github/workflows/e2e-storybook-workflow.yml

```diff
@@ -26,7 +26,13 @@ jobs:
       run: |
         source scripts/e2e-setup-ci.sh
         yarn dlx create-react-app my-cra && cd my-cra
-        yarn add -D eslint-config-react-app
+
+        # TODO: Remove when https://github.com/webpack-contrib/mini-css-extract-plugin/issues/896 is fixed
+        yarn set resolution mini-css-extract-plugin@npm:^2.4.5 2.4.5
```

Comment:
That seems to be fixed (https://github.com/webpack-contrib/mini-css-extract-plugin/issues/896#issuecomment-1014458314)

Reply:
Right, I only removed it from the `create-react-app` e2e workflow, forgot this one 😅